config/mainnet/exceptions.js

```diff
@@ -39,10 +39,6 @@
 module.exports = {
 	blockRewards: [],
 	delegates: [],
-	genesisPublicKey: {
```

Comment:
Why is this been removed? If it is correct, we should remove the description headers in every new exception file like https://github.com/LiskHQ/lisk/blob/dce611af6a5068b8443d6ec84021345baa0259f9/config/devnet/exceptions.js#L25

Reply:
Yes this is removed because of this change https://github.com/LiskHQ/lisk/pull/2149/files#diff-9f30204b3444de4b458d4395cfe3bdf7R493